docs/time_series_for_support_staff.md

```diff
@@ -0,0 +1,161 @@
+Time Series For Devs/Sys-Admins
+===============================
+
+So Riak Time Series has gone live in your organisation? What now? What is it? How do you support it?
+
+You can view this document as a prezzo at:
+http://remarkjs.com/remarkise
+
+---
+
+Overview
+--------
+
+Time Series is a storage and query layer on top of Riak KV that provides new ways of managing data on the normal Riak platform.
+
+It also provides an SQL shell, `riak-shell`, which enables you to configure and query your Time Series system in an intuitive manner.
+
+At its core is a new type of bucket that provides a customisable degree of data colocation.
+
+---
+
+Riak KV Data Location
+---------------------
+
+In Riak KV you write data to a key and it is written at (usually) 3 vnodes on (usually) 3 physical nodes in your cluster. You write data to another key and that is written to a different set of 3 vnodes on a different set of 3 physical nodes.
+
+To query over a Key Value store like Riak KV then you need to scan the entire cluster.
```

Comment:
I find "query" a bit vague here. Since retrieving known keys doesn't require such a scan it'd be good to clarify.